src/Aspire.Hosting/build/Aspire.Hosting.targets

```diff
@@ -34,12 +34,24 @@
     <ItemGroup>
       <AspireProjectMetadataSource Update="@(AspireProjectMetadataSource)">
         <Source>
-        <![CDATA[namespace Projects%3B
-
+        <![CDATA[//------------------------------------------------------------------------------
+// <auto-generated>
+//     This code was generated by a tool.
+//
+//     Changes to this file may cause incorrect behavior and will be lost if
+//     the code is regenerated.
+// </auto-generated>
+//------------------------------------------------------------------------------
+
+namespace Projects%3B
+
+[global::System.CodeDom.Compiler.GeneratedCode("Aspire.Hosting", "]]>$(MicrosoftDotnetSdkInternalPackageVersion)<![CDATA[")]
+[global::System.Diagnostics.CodeAnalysis.ExcludeFromCodeCoverage(Justification = "Generated code.")]
 [global::System.Diagnostics.DebuggerDisplay("Type = {GetType().Name,nq}, ProjectPath = {ProjectPath}")]
 public class ]]>%(ClassName)<![CDATA[ : global::Aspire.Hosting.IProjectMetadata
 {
-  public string ProjectPath => """]]>%(ProjectPath)<![CDATA["""%3B
+    [global::System.CodeDom.Compiler.GeneratedCode("Aspire.Hosting", "]]>$(MicrosoftDotnetSdkInternalPackageVersion)<![CDATA[")]
+    public string ProjectPath { get%3B } = """]]>%(ProjectPath)<![CDATA["""%3B
```

Comment:
Please revert this change. There is no reason to make a field on the object just to store this constant string.